src/server.js

```diff
@@ -75,7 +75,10 @@ server
       cookie: {
         maxAge: 30 * 24 * 60 * 60 * 1000, // 30 days
         sameSite: "strict",
-        secure: !dev && !test,
+        // secure=false because I'm really struggling to get this to work on the server
+        // It's probably something to do with reverse-proxying, and when it's broken no one can log in
+        // Might be worth looking at if relevant when upgrading to SvelteKit
+        secure: false,
```

Comment:
what does secure mean? Is this actually a security problem or just a stressy-sounding name?

Reply:
Requires that the site uses https for cookies, which is not something we currently have anyway. Mostly just a stressy sounding name.